app/views/developers/developers.scala.html

```diff
@@ -101,12 +101,44 @@ <h1 class="flush--top">Developers</h1>
 
     @if(users.length > 0) {
     <div class="form-group>">
-        <p><a class="button" href="mailto:?bcc=@emails">Email @{users.length} developers</a></p>
+        <p>
+            <a class="button" href="mailto:?bcc=@emails" title="Opens external mail client BCC'ing all @{users.length} email addresses.">Email @{users.length} developers</a>
+            <a class="button" href="#" title="Copies all @{users.length} email addresses into the system clipboard." onclick="copyTextToClipboard('@emails'); return false;">Copy to clipboard</a>
+        </p>
     </div>
     }
     <div class="form-group>">
         <a class="back-to-top-link" href="#">Back to Top</a>
     </div>
+
+    <script type="text/javascript">
```

Comment:
Nothing on public; was some stuff on private but it looked like it used some Adobe Flash abomination to do the clipboard copy-paste mechanics